DedicatedServer/Scripts/Libs/Zrx/ModeLibs/Common/Collision/CylinderHitbox.Script.txt

```diff
@@ -239,6 +212,34 @@ Struct_Hitbox GetHitbox(CSmPlayer _Player) {
 	return Struct_Hitbox{Center = Center, Diameter = Diameter, Height = Height};
 }
 
+
+Struct_Hitbox GetSmallHitbox(CSmPlayer _Player) {
```

Comment:
I need some explanation on what's the logic behind this smaller Hitbox.
Also This is a lot of code duplucation, just for a 0.5 factor. Try to put this and the method above together into one method.

```
Struct_Hitbox GetSizedHitbox(CSmPlayer _Player, Real Factor) {
    //...
}

Struct_Hitbox GetHitbox(CSmPlayer_ Player) {
    GetSizedHitbox(Player, 1.);
}
```